PrivateNetworkExample/README.md

```diff
@@ -0,0 +1,13 @@
+# easycontract
```

Comment:
nice name

Reply:
Lets add these files in a directory called `easycontract`

Reply:
**easycontract** Instead of **PrivateNetworkExample** ?

Reply:
Yes